### PR TITLE
Fix rhel 85 appstream pkglist

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: distributions/rhel-85/rhel-85.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8.5/x86_64/baseos/os",
       "rhsm": true
     }, {
-      "id": "baseos",
+      "id": "appstream",
```
Comment (teg) [inline distributions/rhel-85/rhel-85.json:14]: I used this was the bug? For the future, could you explain it in the commit message?

File: internal/distribution/distribution_test.go [hunk 2]
```diff
@@ -56,12 +56,30 @@ func TestAvailableDistributions(t *testing.T) {
...
 
 func TestFindPackages(t *testing.T) {
-	pkgs, err := FindPackages("../../distributions", "centos-8", "x86_64", "ssh")
+	pkgs, err := FindPackages("../../distributions", "centos-8", "x86_64", "vim")
 	require.NoError(t, err)
-	require.Greater(t, len(pkgs), 0)
-	for _, p := range pkgs {
-		require.Contains(t, p.Name, "ssh")
-	}
+	require.ElementsMatch(t, []Package{
```
Comment (teg) [inline internal/distribution/distribution_test.go:61]: Makes sense, but explanation in the commit message would make it perfecter.

**Review by teg — APPROVED** [review]
Looks good, minor nits inline not a blocker.

File: distributions/rhel-85/rhel-85.json (hunk 1 above)
Comment (Gundersanne) [reply distributions/rhel-85/rhel-85.json:14]: done!

File: internal/distribution/distribution_test.go (hunk 2 above)
Comment (Gundersanne) [reply internal/distribution/distribution_test.go:61]: done!

**Pull request merged**